### PR TITLE
Fix: bootstrap: raise warning when configuring diskless SBD with less than 3 nodes

### PR DESCRIPTION
**Pull request by liangxin1300**
## Problem
Using diskless SBD with less than 3 nodes was not recommend, it'd better if bootstrap giving related warning while configuring diskless SBD

### CODE REVIEW

File: crmsh/bootstrap.py [hunk 1]
```diff
@@ -525,6 +527,8 @@ def sbd_init(self):
         if not self._sbd_devices and not self.diskless_sbd:
             invoke("systemctl disable sbd.service")
             return
+        if self.diskless_sbd:
+            warn(self.DISKLESS_SBD_WARNING)
```
Comment (liangxin1300) [inline crmsh/bootstrap.py:531]: Always give warning while configuring diskless SBD on init process

File: crmsh/bootstrap.py [hunk 2]
```diff
@@ -563,6 +567,10 @@ def join_sbd(self, peer_host):
         dev_list = self._get_sbd_device_from_config()
         if dev_list:
             self._verify_sbd_device(dev_list, [peer_host])
+        else:
+            vote_dict = utils.get_quorum_votes_dict(peer_host)
+            if int(vote_dict['Total']) < 2:
+                warn(self.DISKLESS_SBD_WARNING)
```
Comment (liangxin1300) [inline crmsh/bootstrap.py:573]: On join side, when finding that init node's actual votes less than 2, give warning

Comment (gao-yan) [reply crmsh/bootstrap.py:573]: It seems to make more sense to check the "Expected votes", which means whether the "configuration"/"setup" is going to meet the requirement.

**Review by gao-yan — COMMENTED** [review]
Helpful warning. Thanks.

File: crmsh/bootstrap.py (hunk 2 above)
Comment (liangxin1300) [reply crmsh/bootstrap.py:573]: Changed, thanks!

**Review by gao-yan — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**